sdk/src/connections/target/addi9036_sensor.cpp

```diff
@@ -160,6 +161,8 @@ aditof::Status Addi9036Sensor::open() {
     }
 
     m_implData->numVideoDevs = driverSubPaths.size();
+
+    assert(m_implData->numVideoDevs > 0);
```

Comment:
Not sure if this is needed as having no videoDevs is not a critical error, it might occur in normal usage (if no cameras are connected) and shall not crash the application